src/wymeditor/editor/dialogs.js

```diff
@@ -1,5 +1,26 @@
 "use strict";
 /**
+    editor._openPopupWindow
+    =======================
+
+    Opens a popup window, provided the `windowFeatures`.
+*/
+WYMeditor.editor.prototype._openPopupWindow = function (windowFeatures) {
+    var wym = this,
+        popup,
+        basePath = wym._options.basePath;
+
+    popup = window.open(
+        // https://github.com/wymeditor/wymeditor/issues/731
+        jQuery.browser.msie ? basePath + 'popup.html' : '',
```

Comment:
All other browsers do no have this issue thus are fine with a blank string here.